turbinia/workers/analysis/jupyter_test.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import unittest
+
+from turbinia import config
+from turbinia.workers.analysis import jupyter
+
+
+class JupyterAnalysisTaskTest(unittest.TestCase):
+  """test for the Jupyter notebook analysis task."""
+
+  BAD_JUPYTER_CONFIG = """c.NotebookApp.token = ''
+c.NotebookApp.password = ''
+c.NotebookApp.open_browser = False
+c.NotebookApp.port = 0
+c.NotebookApp.allow_origin_pat = '(^https://8080-dot-[0-9]+-dot-devshell\.appspot\.com$)z]*.notebooks.googleusercontent.com)'
+c.NotebookApp.allow_remote_a’ccess = True
```

Comment:
Couple quick things here.  I guess this single quote is just an accident, but doesn't really affect anything?  Also, I am thinking checking for `allow_remote_access = True` is probably a good thing to add to the report output?

Reply:
I can add this check but this is true by default and I think it can add too much noise to the report.